torch/csrc/jit/codegen/cuda/index_reference_replay.cpp

```diff
@@ -261,8 +265,11 @@ IndexCompute getReferenceIndexing(
     auto lowered_id = gpu_lower->lowerValue(reference_tensor->axis(loop_i))
                           ->as<kir::IterDomain>();
     initial_index_map[lowered_id] = loop_structure[loop_i]->index();
-    if (loop_structure[loop_i]->iter_domain()->parallelType() ==
-        ParallelType::Vectorize) {
+    if (loop_structure[loop_i]->vectorize() &&
+        (loop_structure[loop_i]->iter_domain()->parallelType() ==
+             ParallelType::Vectorize ||
+         loop_structure[loop_i]->iter_domain()->parallelType() ==
+             ParallelType::MisalignedVectorize)) {
```

Comment:
When `vectorize()` is false but `iter_domain()->parallelType()` is either `Vectorize` or `MisalignedVectorize`?

Reply:
It isn't.